src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -255,13 +262,6 @@ public boolean isFLAC() {
 	}
 
 	/**
-	 * @return True if the audio codec is HE-AAC.
-	 */
-	public boolean isHEAAC() {
-		return FormatConfiguration.AAC_HE.equalsIgnoreCase(getCodecA());
```

Comment:
Why do you want i keep it ?
isHEAAC()
isAACHE()
is AAHEC()...
How many do you need ? for what purpose ?